docs/src/index.md

```diff
@@ -341,32 +341,62 @@ To get in touch with the developers,
+  ERC      --><img align="middle" src="https://github.com/trixi-framework/Trixi.jl/assets/3637659/9371e7e4-3491-4433-ac5f-b3bfb215f5ca" height="120"><!--
+  NSF      --><img align="middle" src="https://github.com/trixi-framework/Trixi.jl/assets/3637659/3d07b627-c602-483d-93c6-19ac88bba125" height="120"><!--
+  DUBS     --><img align="middle" src="https://github.com/trixi-framework/Trixi.jl/assets/3637659/c7f0ea4d-9865-40f4-b0bd-9f4709356f22" height="120"><!--
+  NumFOCUS --><img align="middle" src="https://github.com/trixi-framework/Trixi.jl/assets/3637659/8496ac9e-b586-475f-adb7-69bcfc415185" height="120"><!--
+  -->
+</p>
+```
 
-This project has benefited from funding by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation)
-through the research unit FOR 5409 "Structure-Preserving Numerical Methods for Bulk- and
-Interface Coupling of Heterogeneous Models (SNuBIC)" (project number 463312734).
+This project has benefited from funding by the [Deutsche
+Forschungsgemeinschaft](https://www.dfg.de/) (DFG, German Research Foundation)
+through the following grants:
+* Excellence Strategy EXC 2044-390685587, Mathematics Münster: Dynamics-Geometry-Structure.
+* Research unit FOR 5409 "Structure-Preserving Numerical Methods for Bulk- and
+  Interface Coupling of Heterogeneous Models (SNuBIC)" (project number 463312734).
+* Individual grant no. 528753982.
 
-This project has benefited from funding from the European Research Council through the
+This project has benefited from funding from the [European Research Council](https://erc.europa.eu)
+through the
 ERC Starting Grant "An Exascale aware and Un-crashable Space-Time-Adaptive
 Discontinuous Spectral Element Solver for Non-Linear Conservation Laws" (Extreme),
 ERC grant agreement no. 714487.
 
-This project has benefited from funding from Vetenskapsrådet (VR, Swedish Research Council), Sweden
+This project has benefited from funding from [Vetenskapsrådet](https://www.vr.se)
+(VR, Swedish Research Council), Sweden
 through the VR Starting Grant "Shallow water flows including sediment transport and morphodynamics",
 VR grant agreement 2020-03642 VR.
 
-This project has benefited from funding from the United States National Science Foundation under awards
+This project has benefited from funding from the United States
+[National Science Foundation](https://www.nsf.gov/) (NSF) under awards
 DMS-1719818 and DMS-1943186.
 
-This project has benefited from funding from the German Federal Ministry of
-Education and Research through the project grant "Adaptive earth system modeling
+This project has benefited from funding from the German
+[Federal Ministry of Education and Research](https://www.bmbf.de) (BMBF)
+through the project grant "Adaptive earth system modeling
 with significantly reduced computation time for exascale supercomputers
 (ADAPTEX)" (funding id: 16ME0668K).
+
+Trixi.jl is supported by [NumFOCUS](https://numfocus.org/) as an Affiliated Project.
```

Comment:
```suggestion

This project has benefited from funding from the 
[Daimler and Benz foundation](https://www.daimler-benz-stiftung.de/cms/en/)
(project number 32-10/22).

Trixi.jl is supported by [NumFOCUS](https://numfocus.org/) as an Affiliated Project.
```

Reply:
Done in https://github.com/trixi-framework/Trixi.jl/pull/1697/commits/06649b261a005bb19f298809e4c3cd477cbabb0e.